src/lmo_wrapper.jl

```diff
@@ -0,0 +1,179 @@
+"""
+    BLMO
+
+Supertype for the Bounded Linear Minimization Oracles
+"""
+abstract type BoundedLinearMinimizationOracle <: FrankWolfe.LinearMinimizationOracle end 
+
+###################################### Necessary to implement ####################################
+
+"""
+Implement `FrankWolfe.compute_extreme_point`
+
+Given a direction d solves the problem
+    min_x d^T x
+where x has to be an integer feasible point
+"""
+function compute_extreme_point end
```

Comment:
I don't think having the empty function is needed, since we are already importing it?